### PR TITLE
logger: initialize inspirehep-logger

### PR DESCRIPTION
**Pull request by drjova**
* Use ``sentry-sdk`` instead of ``raven``.

* Drop ``invenio-logging``.

* INSPIR-2429

Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: inspirehep/logger/ext.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class InspireLogger:
+    def __init__(self, app=None):
+        if app:
+            self.init_app(app)
+
+    def init_app(self, app):
+        sentry_dsn = app.config.get("SENTRY_DSN")
+        if sentry_dsn is not None:
```
Comment (ammirate) [inline inspirehep/logger/ext.py]: `if sentry_dsn:`
:)

**Comment by drjova** [comment]
let's wait until we have labs on k8s

**Comment by ammirate** [comment]
@drjova I think we can merge this now.

File: inspirehep/logger/config.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+dictConfig(
+    {
+        "version": 1,
+        "formatters": {
+            "default": {
+                "format": "[%(asctime)s: %(levelname)s/%(processName)s] %(name)s: %(message)s"
+            }
+        },
+        "handlers": {
+            "stdout_handler": {"class": "logging.StreamHandler", "formatter": "default"}
```
Comment (michamos) [inline inspirehep/logger/config.py]: don't we want to log more on stdout (like WARNING/INFO)?

Comment (drjova) [reply inspirehep/logger/config.py]: updated

Comment (drjova) [reply inspirehep/logger/config.py]: is it what you meant? about the ``level``?

Comment (michamos) [reply inspirehep/logger/config.py]: yeah, but don't you need to set the priority fo sentry too? or is that automatic?

Comment (drjova) [reply inspirehep/logger/config.py]: it's automatic for sentry

**Pull request merged**